projects/core/directives/items-handlers/items-handlers.directive.ts

```diff
@@ -0,0 +1,51 @@
+import {Directive, inject, Input, signal} from '@angular/core';
+import type {
+    TuiBooleanHandler,
+    TuiIdentityMatcher,
+    TuiStringHandler,
+} from '@taiga-ui/cdk/types';
+
+import type {TuiItemsHandlers} from './items-handlers.tokens';
+import {TUI_ITEMS_HANDLERS} from './items-handlers.tokens';
+
+@Directive({
+    standalone: true,
```

Comment:
Should it provide itself as `TUI_ITEMS_HANDLERS`? Looks like components should work with `TUI_ITEMS_HANDLERS` and not a particular directive implementation. Like with other cases like dropdown/hint options, textfield options etc.

Reply:
https://github.com/taiga-family/taiga-ui/pull/10642/commits/aee9541a2f99a35138eb6a87857842265793ed92